### PR TITLE
Update email setup instructions

### PR DESCRIPTION
**Pull request by coreyhulen**

### CODE REVIEW

File: doc/install/SMTP-Email-Setup.md [hunk 1]
```diff
@@ -29,15 +30,42 @@ To enable email, configure an SMTP email service as follows:
...
 	       10. **Connection Security**: `TLS (Recommended)`
 	       11. Then click **Save**
+	       12. Then click **Test Connection**
+	       13. If the test failed please look in **OTHER** > **Logs** for any errors that look like `[EROR] /api/v1/admin/test_email ...`
+
+### Known Good Sample Settings
+
+##### Amazon SES
+* Set **SMTP Username** to **AKIASKLDSKDIWEOWE**
+* Set **SMTP Password** to **AdskfjAKLSDJShflsdfjkakldADkjkjdfKAJDSlkjweiqQIWEOU**
```
Comment (crspeller) [inline doc/install/SMTP-Email-Setup.md]: Do these credentials work?

File: doc/install/SMTP-Email-Setup.md [hunk 2]
```diff
@@ -29,15 +30,42 @@ To enable email, configure an SMTP email service as follows:
...
+* Set **SMTP Server** to **email-smtp.us-east-1.amazonaws.com**
+* Set **SMTP Port** to **465**
+* Set **Connection Security** to **TLS**
+
+##### Postfix
+* Make sure Postfix is installed on the machine where Mattermost is installed
```
Comment (crspeller) [inline doc/install/SMTP-Email-Setup.md]: This probably won't work for sending email externally.

File: doc/install/SMTP-Email-Setup.md (hunk 1 above)
Comment (coreyhulen) [inline doc/install/SMTP-Email-Setup.md]: nope they are junk

File: doc/install/SMTP-Email-Setup.md (hunk 2 above)
Comment (coreyhulen) [inline doc/install/SMTP-Email-Setup.md]: why not?  this is how docker was originally configured?

**Pull request merged**